buddyboss/buddyboss-platform
# Backend "View Tutorial" buttons should be move to top

### All Submissions:



Fixes #2197 

### Connected PR
https://github.com/buddyboss/buddyboss-platform-pro/pull/151

### How to test the changes in this Pull Request:

1. Check All backend settings which have "View Tutorial" Button
2. It should be positioned at the top, besides heading.

### Proof Screenshots or Video
https://www.loom.com/share/92049c8e220f4d23ac301765b0dff54f

### Other information:
